core/src/main/resources/static/media/img/en_GB/app-store-badge.svg

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
When I use `Locale.setDefault(Translator.EN_US_LOCALE);` in the `setUp` it works.